lib/hacluster.pm

```diff
@@ -220,6 +223,10 @@ sub post_fail_hook {
 }
 
 sub test_flags {
+    # 'fatal'          - abort whole test suite if this fails (and set overall state 'failed')
+    # 'ignore_failure' - if this module fails, it will not affect the overall result at all
+    # 'milestone'      - after this test succeeds, update 'lastgood'
+    # 'norollback'     - don't roll back to 'lastgood' snapshot if this fails
```

Comment:
better don't add it. We don't want to have this description in *all* testmodules so better have it in *none*